src/modules/involapi.js

```diff
@@ -0,0 +1,107 @@
+// This module is made to handle the Involvement Api Requests.
+
+// involapiUrl = Base Url
+const involapiUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/';
+
+// These IDs are for testing purposes.
+// const idApp = '1nKEgSDFUEKqARuhDNsk';
+// const idApp = '01g8zOGHAMdlxh7XvGhS';
+const idApp = '0CG34aiKZpxDXV0UfZ4U';
+// const idApp = 'ZYPqezmGTKMkv14Q0GGV'; // failed
+
+// Next ID is for production.
+// const idApp = '7N9OQ8SKImUJfeBFIgcg';
+
+// EndPoints
+// apps/ POST to create a new App >> return :appid
+const involapiApps = 'apps/';
+const involapiFetchUrl = `${involapiUrl}${involapiApps}`;
+
+// RESERVATIONS
+// :app_id/reservations/
+//  GET retrieve the reservations for a Pokemon
+//     /apps/:app_id/reservations?item_id=item1
+//   result:
+//     Array of objects
```

Comment:
[OPTIONAL] 👍🏽 
- I'd like to kindly suggest that you take out all commented lines of code in order to make your code base look cleaner and readable. 🤗

Reply:
Thank you for your suggestions @Favourezeugwa 👍